cmd/reloader/main.go

```diff
@@ -7,12 +7,26 @@ import (
 	"log"
 	"os"
 	"os/signal"
+	"strings"
 	"syscall"
 
-	"github.com/nats-io/nats-operator/pkg/reloader"
+	natsreloader "github.com/nats-io/nats-operator/pkg/reloader"
 	"github.com/nats-io/nats-operator/version"
 )
 
+// StringSet is a wrapper for []string to allow using it with the flags package.
+type StringSet []string
+
+func (s StringSet) String() string {
+	return strings.Join([]string(s), ", ")
+}
+
+// Set appends the value provided to the list of strings.
+func (s StringSet) Set(val string) error {
+	s = append(s, val)
+	return nil
+}
+
```

Comment:
This is not working for me as it is...

```console
go run cmd/reloader/main.go -c /tmp/the-nats-conf/nats.conf -pid /tmp/nats.pid
2019/04/13 11:47:50 Starting NATS Server Reloader v0.4.4-v1alpha2+git
Error: lstat /etc/nats: no such file or directory
exit status 1
```

Looks like they have to be pointer receivers for it to work:

```
Or you can create custom flags that satisfy the Value interface (with pointer receivers) and couple them to flag parsing by

flag.Var(&flagVal, "name", "help message for flagname")
```

I changed this into the following and now works:

```go
// StringSet is a wrapper for []string to allow using it with the flags package.
type StringSet []string

func (s *StringSet) String() string {
	return strings.Join([]string(*s), ", ")
}

// Set appends the value provided to the list of strings.
func (s *StringSet) Set(val string) error {
	*s = append(*s, val)
	return nil
}
```

```
go run cmd/reloader/main.go -c /tmp/the-nats-conf/nats.conf -c /tmp/the-nats-conf/authconfig/auth.conf -pid /tmp/nats.pid
[/tmp/the-nats-conf/nats.conf /tmp/the-nats-conf/authconfig/auth.conf]
2019/04/13 13:29:29 Starting NATS Server Reloader v0.4.4-v1alpha2+git
2019/04/13 13:29:31 Event: "/tmp/the-nats-conf/.#nats.conf": CREATE 
2019/04/13 13:29:32 Event: "/tmp/the-nats-conf/nats.conf": CHMOD 
2019/04/13 13:29:32 Event: "/tmp/the-nats-conf/nats.conf": WRITE 
2019/04/13 13:29:32 Sending signal to server to reload configuration
2019/04/13 13:29:45 Event: "/tmp/the-nats-conf/authconfig/.#auth.conf": CREATE 
2019/04/13 13:29:47 Event: "/tmp/the-nats-conf/authconfig/auth.conf": CHMOD 
2019/04/13 13:29:47 Event: "/tmp/the-nats-conf/authconfig/auth.conf": WRITE 
```

Reply:
Derp yeah, obviously.  My bad.